src/js/api.js

```diff
@@ -2,10 +2,19 @@ const defaultBaseUrl = process.env.DEFAULT_API_URL
   ? `${process.env.DEFAULT_API_URL}`
   : "https://muni-portal-backend.openup.org.za";
 
-function getBaseApiUrl() {
+export function getBaseApiUrl() {
   return sessionStorage.getItem("apiBaseUrl") || defaultBaseUrl;
 }
 
+// import { createServer } from "miragejs"
+
+// createServer({
+//   routes() {
+//     this.post("https://muni-portal-backend.openup.org.za/accounts/login/", () => {detail:"Login or password invalid."})
+//   },
+// })
+
+
```

Comment:
please make sure all commented code is removed when you're ready for a proper review